wcc-contentful-app/app/helpers/wcc/contentful/app/section_helper.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'redcarpet'
+require '../wcc-contentful-app/lib/wcc/contentful/app/markdown_renderer'
```

Comment:
I don't think you need this require anymore because we are requiring it in the main file